### DISCUSSION TITLE
Musikmodul

### DISCUSSION THREAD

**Discussion by IGNOR90** [rafaelurben/django-discordbot#198] [Q&A]
Was sind memes

    os.path.dirname(os.path.realpath(__file__)), "botfiles"))
MEMESPATH = _s("MEMESPATH", os.path.join(FILESPATH, "memes"))
FFMPEGPATH = _s("FFMPEGPATH", os.path.join(FILESPATH, "ffmpeg.exe"))

**Comment by rafaelurben** [comment]
Hey @IGNOR90 

Dies ist ein etwas älteres Feature und ich bin mir nicht sicher, ob es noch läuft. Seit der Migration zu discord.py v2.0 habe ich die Audio-Funktionen nicht mehr getestet.

Auf jeden Fall war es möglich, im `django-discordbot/botfiles/memes/`-Ordner (muss zuerst erstellt werden) diverse kurze `.mp3`-Dateien zu speichern. Diese konnten dann mit dem `/meme <name>`-Befehl im aktuellen Sprachkanal abgespielt werden.

Ich hoffe, ich konnte dir damit helfen ;)

**Reply by rafaelurben** [reply]
@IGNOR90 UPDATE:

Ich habe das Modul getestet - es funktioniert tatsächlich noch, muss jedoch in der config oder mit einer Umgebungsvariable zuerst aktiviert werden. (es ist standardmässig deaktiviert)

https://github.com/rafaelurben/django-discordbot/blob/9c928172b502157408c1e17ac72a167778a12556/discordbot/config.py#L50

**Reply by IGNOR90** [reply]
Ja danke ich verstehe.
Aber jetzt gibt es ein anderes Problem

/play https://youtu.be/tzWggCnYVWU

Gibt eine Fehlermeldung
`Exception ignored in: <function AudioSource.__del__ at 0x000002167F8F9C60>
Traceback (most recent call last):
  File "E:\PycharmProjects\discordbot\venv\lib\site-packages\discord\player.py", line 117, in __del__
    self.cleanup()
  File "E:\PycharmProjects\discordbot\venv\lib\site-packages\discord\player.py", line 628, in cleanup
    self.original.cleanup()
AttributeError: 'YouTubePlayer' object has no attribute 'original'`

**Reply by IGNOR90** [reply]
Ich kann nicht herausfinden, was das Problem ist

**Reply by rafaelurben** [reply]
@IGNOR90 Den `/play`-Befehl habe ich nicht getestet. Ich werde dies noch tun und den Fehler suchen 👍

**Reply by IGNOR90** [reply]
> @IGNOR90Den `/play`-Befehl habe ich nicht getestet. Ich werde dies noch tun und den Fehlersuchen👍
Vielen Dank

**Reply by rafaelurben** [reply]
@IGNOR90 Ich konnte den Fehler in über 15 Versuchen nur ein einziges Mal reproduzieren. Dieser Fehler sollte soweit ich das sehe die Wiedergabe nicht einschränken.

**Comment by IGNOR90** [comment]
Erzähl es mir nicht? Bitte

https://github.com/rafaelurben/django-discordbot/discussions/198#discussioncomment-3967575

пн, 24 окт. 2022 г. в 20:29, Rafael ***@***.***>:
